Source/Core/DolphinQt/Config/InfoWidget.cpp

```diff
@@ -25,6 +26,8 @@
 
 InfoWidget::InfoWidget(const UICommon::GameFile& game) : m_game(game)
 {
+  m_volume = DiscIO::CreateVolume(m_game.GetFilePath());
```

Comment:
I wasn't sure if it was worth invalidating game list caches and adding the TMD to GameFile, so I opted for a simpler method: open the volume ourselves in InfoWidget.